### ISSUE TITLE
XenServer 7 has new name for guest extensions CD

### ISSUE THREAD

**Issue by wkoszek** [xenserver/packer-builder-xenserver#83]
Was: xs-tools.iso

Is: guest-tools.iso

**Comment by gaserre** [comment]
in case you didn't find it, theres a 

"tools_iso_name": "guest-tools.iso" 

that you can set in the .json file.

**Comment by wkoszek** [comment]
@gaserre Yes, I saw that. But my first invocation was unsuccessful. Wondering if we could just add additional "if" to save people time in searching for this stuff.

**Comment by rprots** [comment]
Error is shown:

`==> xenserver-iso: Couldn't find a VDI named 'xs-tools.iso'`

(adding for search).
